packages/core/admin/ee/admin/pages/SettingsPage/pages/ReviewWorkflows/pages/ListView/ListView.js

```diff
@@ -90,6 +92,14 @@ export function ReviewWorkflowsListView() {
     }
   );
 
+  const getContentTypeDisplayName = (uid) => {
+    const contentType = [...collectionTypes, ...singleTypes].find(
+      (contentType) => contentType.uid === uid
+    );
+
+    return contentType.info.displayName;
```

Comment:
I think we should check that we have the content-type, only because TS would want us to as well 👍🏼

Reply:
We must have it, otherwise a user can't select it. There is no way (except manual editing a DB entry) this doesn't exist.

Reply:
If you're certain, when converting to TS we'll have to do a `non-null assertion`